docs/framework/unmanaged-api/debugging/debugging-interfaces.md

```diff
@@ -296,7 +296,10 @@ This section describes the unmanaged interfaces that handle the debugging of a p
  A logical extension of `ICorDebugProcess`.  
   
  [ICorDebugProcess3 Interface](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess3-interface.md)  
- Controls custom debugger notifications.  
+ Controls custom debugger notifications.
+
+ [ICorDebugProcess4 Interface](../../../../docs/framework/unmanaged-api/debugging/icordebugprocess4-interface.md)
+ Provides support for out of process execution control.
   
```

Comment:
the same problem occurs for the other new APIs that were recently added at the bottom of this article